### ISSUE TITLE
"library(rgee)" does not work in R studio but works in R

### ISSUE THREAD

**Issue by gc759** [r-spatial/rgee#189]
* rgee version:1.1.0
* R version:4.1.1
* Operating System:macOS

#### At submit an issue, please attached the following information of your `rgee` session:

- [ ] You have the Python API installed (from terminal):
```
earthengine -h
```

<img width="341" alt="截屏2021-10-26 23 54 46" src="https://user-images.githubusercontent.com/71622914/138972476-4620ad29-5119-470d-a15d-9b6be62e8790.png">

Attach your Python (reticulate) configuration:

```r
library(reticulate)
py_config()
```
<img width="867" alt="截屏2021-10-26 23 56 18" src="https://user-images.githubusercontent.com/71622914/138972639-204633b9-70b7-4c38-a4b8-49e2eb3bf5f3.png">

### Description

When I run "library(rgee)" in RStudio, it gives back the following error message:

<img width="810" alt="截屏2021-10-26 23 52 38" src="https://user-images.githubusercontent.com/71622914/138972302-ba4fc651-3a34-4a0a-86b3-20f53ab4f001.png">

However, it works well in R, and ee_check() gives a nice feedback as well.

<img width="389" alt="截屏2021-10-26 23 59 16" src="https://user-images.githubusercontent.com/71622914/138972910-ecb8d24d-fc7e-49e0-b5c0-566eb0603929.png">

### What I Did

I believe "library(rgee)" used to work well in RStusio, until I used the following codes to install EE API manually as suggested in #91 :

<img width="912" alt="截屏2021-10-27 00 01 34" src="https://user-images.githubusercontent.com/71622914/138973154-6e89c8b9-e92f-497d-ab31-9aef6f93cf51.png">

I hope you can help to let me use library(rgee) in RStudio.

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by csaybar** [comment]
Hi @gc759 sorry for the late reply.

1) Did you try re-installiing rgee?
https://stackoverflow.com/questions/67001134/am-not-able-to-load-tidyverse-in-r-4-0-5-any-leads

2) Are you sure that you are using the same version of R in Rstudio?. Please run in both Rstudio and your R console this:

```r
R.Version()
```
In mac, you do not need more than `ee_install()` to set properly the Python ENV that rgee needs.

**Comment by gc759** [comment]
Hi @csaybar ,
Thank you for the suggestions! 

I tried to remove and reinstall rgee, but was unable to install it again. Maybe it's because I didn't manage to remove the package competely, but I'm not sure.

I formatted my laptop and started from the first step, but after succeeding (passed the ee_check and did some work with Google earth engine) once, my "earthengine-api" still disappeared again (ee_check shows that it's not installed). Then I used ee_install_upgrade() trying to get earthengine-api back, but even though it tells me it is installed, the ee_check() still shows that it was not installed( and I can work with google earth engine no more).

<img width="558" alt="截屏2021-10-27 22 05 14" src="https://user-images.githubusercontent.com/71622914/139559583-70f4af8b-1cb2-4e1c-8862-3ba37689d101.png">

I eventually used "pip install earthengine-api" in my terminal to get it, and this time I succeeded. I do hope that it won't disappear again.
<img width="226" alt="Screenshot 2021-10-28 at 14 27 34" src="https://user-images.githubusercontent.com/71622914/139559588-908a1e68-6cce-4698-b34a-c91d4734d610.png">

I'm still wondering why I kept losing "earthengine-api", but at least my problem is solved for now.

Also, regarding to R.Version(), R and RStudio are giving me the same out puts.

R:
<img width="503" alt="Screenshot 2021-10-30 at 23 06 53" src="https://user-images.githubusercontent.com/71622914/139559637-5da25018-ea8d-4dc0-b76b-3bae9e8cd4ee.png">

RStudio
<img width="423" alt="Screenshot 2021-10-30 at 23 06 40" src="https://user-images.githubusercontent.com/71622914/139559645-3bd2dc63-4167-4ae4-967d-5f30e63561a9.png">
